source/getting-started/configuration.rst

```diff
@@ -1,9 +1,119 @@
+
+.. code:: bash
+
+    remote-control:
+        # enable remote-control
+        control-enable: yes
+
+        # location of the files created by unbound-control-setup
+        #server-key-file: "/usr/local/etc/unbound/unbound_server.key"
+        #server-cert-file: "/usr/local/etc/unbound/unbound_server.pem"
+        #control-key-file: "/usr/local/etc/unbound/unbound_control.key"
+        #control-cert-file: "/usr/local/etc/unbound/unbound_control.pem"
+
+To set up for using the :command:`unbound-control` command, we need to invoke the :command:`unbound-control-setup` command. This creates a number of files in the default install director directory. The default install directory is ``/usr/local/etc/unbound/unbound.conf`` on most systems, but some distributions may put it in ``/etc/unbound/unbound.conf`` or ``/etc/unbound.conf``.
+
+Apart from an extensive config file, with just about all the possible configuration options, :command:`unbound-control-setup` creates the cryptographic keys necessary for the control option. 
+
+.. code:: bash
+
+    unbound-control-setup
+
+If you use a username like ``unbound`` in the config to run the daemon, you can use sudo to create the file in that user's name, so that the server is allowed to read the keys. This also works for other users if the ``/usr/local/etc/unbound/`` directory is write-protected.
+
+.. code:: bash
+
+	``sudo -u unbound unbound-control-setup``
+
+When these steps succeed, you can now control Unbound using the :command:`unbound-control` command. Note that if you're not using the name ``unbound.conf`` in the default directory, the name (and possibly path) need to be provided when using the command using the :option:`-c` flag.
+
+Set up trust anchor
+===================
+
+To enable `DNSSEC <https://en.wikipedia.org/wiki/Domain_Name_System_Security_Extensions>`_, which we strongly recommend, we need to create a trust anchor as it ensures the integrity of the responses to your queries.
+
+To help, we can use the :command:`unbound-anchor` command. :command:`unbound-anchor` performs the setup by creating a root key. The default location that :command:`unbound-anchor` creates this in is determined by your installation method. Usualy the default directory is ``/usr/local/etc/unbound/``.
+
+.. code::bash
+
+	unbound-anchor
+
+Note that using a package manager to install Unbound, on some distributions, creates the root key during installation. On Ubuntu 20.04.1 LTS for example, this location is ``/var/lib/unbound/root.key``. On macOS Big Sur this location is ``/opt/homebrew/etc/unbound/root.key`` If you create the root key yourself (by using the :command:`unbound-anchor` command), then the file path in the config should be changed in the config to the default location. To find out the default location you can use the :command:`unbound-anchor` command again with the ``-vvv`` option enabled.
```

Comment:
Duplication. Perhaps reduce "then the file path in the config should be changed in the config to the default location" to "then the path to the anchor file in the configuration file should be changed to the correct location" ?